src/server/server_family.cc

```diff
@@ -1146,6 +1146,17 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
     }
   }
 
+  {
+    const auto& reply_stats = m.facade_stats.reply_stats;
+    string sript_error_type_metric;
```

Comment:
Sorry, I think it's unnecessary detailed. The goal is to be able to get alerted about servers that have an error and then go to the server logs and check the warnings. So I think a simple `script_error` counter (not GAUGE) should do the job here.

Reply:
ok fixed